core-bundle/src/Twig/Extension/ContaoExtension.php

```diff
@@ -358,6 +360,11 @@ public function getFilters(): array
                 [SanitizerRuntime::class, 'sanitizeHtml'],
                 ['is_safe' => ['html']],
             ),
+            new TwigFilter(
+                'allow_html',
+                [SanitizerRuntime::class, 'allowHtml'],
+                ['is_safe' => ['html']],
+            ),
```

Comment:
We might want to use our own sanitizer implementation like `|sanitize_html('contao')` here to better fit the “Symfony way”.

Reply:
Is `contao` the right name? Or should we name it `backend` or something like that to emphasize where the data is coming from or why we want to sanitize it differently? Or `rich_text`?

Reply:
IMHO it has nothing to do with the rich text fields (there you would not even need the filter) but is the back end wide setting that affects all fields with `allowHtml`. I currently think I would rather go with something like `contao` or `contao_backend`.